gradle/libs.versions.toml

```diff
@@ -103,7 +103,6 @@ accompanist-pager-indicators = { module = "com.google.accompanist:accompanist-pa
 firebase-bom = { module = "com.google.firebase:firebase-bom", version.ref = "firebase" }
 firebase-auth = { module = "com.google.firebase:firebase-auth-ktx" }
 firebase-authUI = "com.firebaseui:firebase-ui-auth:7.2.0"
-firebase-dynamicLink = { module = "com.google.firebase:firebase-dynamic-links-ktx" }
 
```

Comment:
Removed the Firebase Dynamic Links dependency. This change is necessary as we're transitioning from Firebase Dynamic Links to Airbridge for deep linking functionality. The Airbridge SDK documentation recommends removing Firebase Dynamic Links when implementing Airbridge deep linking to avoid conflicts.